waku/common/databases/db_postgres/dbconn.nim

```diff
@@ -300,18 +302,19 @@ proc dbConnQueryPrepared*(
     error "error in dbConnQueryPrepared", error = $error
     return err("error in dbConnQueryPrepared calling sendQuery: " & $error)
 
+  let stmtNameSummary = stmtName[0 ..< min(stmtName.len, 128)]
```

Comment:
This is incorrect. The issue isn't length but cardinality of labels. We need another fix for these metrics.

No metrics should set values for labels that are unbound.

Reply:
Please read: https://prometheus.io/docs/practices/naming/#labels